utils/logger.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const LogRefKey = "reference"
+
+//LogEntryWithRef returns a logrus Entry with a random unique value for requestId field
+func LogEntryWithRef() *logrus.Entry {
+	return logrus.WithField(LogRefKey, time.Now().Unix())
```

Comment:
Wondering why we always log "reference"
We could pass logrefkey to the function, perhaps? 🤔

Reply:
I believe it is for standardize log message (like maintaining constant messages)...if we provide privilege to pass logrefkey to the function, then developer will define their own reference for logging..